tensorly/decomposition/_parafac2.py

```diff
@@ -364,7 +370,10 @@ def parafac_updates(X, w, f):
 
         if tol:
             rec_error = _parafac2_reconstruction_error(
-                tensor_slices, (weights, factors, projections), norm_tensor
+                tensor_slices,
+                (weights, factors, projections),
+                norm_tensor,
```

Comment:
I saw the black formatting test failed. Run `black tensorly` to reformat everything. I don't know where the actual formatting issue is—this is just a place to throw the comment.